pyproject.toml

```diff
@@ -197,7 +201,9 @@ extend-ignore = [
 
 [tool.ruff.per-file-ignores]
 "tests/*.py" = ["D", "E501"]
-"examples/*.py" = ["D", "B"]
+"docs/examples/*.py" = ["D", "B"]
+"docs/examples/napari/*" = ["E501"]
```

Comment:
Turns out the inline `noqa`statements were being rendered as text in the actual website. This seems like a good way to avoid that problem.